src/EventBuilder/Cecil/EventTemplateInformation.cs

```diff
@@ -43,7 +65,12 @@ private static string GetEventArgsTypeForEvent(EventDefinition ei)
                         type.GenericParameters.Zip(generic.GenericArguments, (name, actual) => new { name, actual })) {
                     var realType = GetRealTypeName(kvp.actual);
 
-                    ret = ret.Replace(kvp.name.FullName, realType);
+                    var temp = ret.Replace(kvp.name.FullName, realType);
```

Comment:
I'm not sure I quite get this change, shouldn't we be replacing type names for multiple generic type arguments?